### PR TITLE
SPLAT-1116: vsphere-build02

### PR DESCRIPTION
**Pull request by jcpowermac**

### CODE REVIEW

**Comment by jcpowermac** [comment]
```
cd to_directory_release
git pull --rebase upstream master
git checkout -b vsphere-build02
cd to_directory_ci_tools
git pull --rebase upstream master
go run ./cmd/cluster-init --create-pr=false --release-repo ../release --cluster-name vsphere-build02
go run ./cmd/cluster-init --create-pr=false --release-repo ../release --cluster-name vsphere-build02 --update
cd to_directory_release
make update
```

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -2144,6 +2147,12 @@ secret_configs:
     sa.pod-scaler.multi01.token.txt:
       field: sa.pod-scaler.multi01.token.txt
       item: pod-scaler
+    sa.pod-scaler.vsphere-build02.token.txt:
```
Comment (bear-redhat) [inline core-services/ci-secret-bootstrap/_config.yaml:2150]: remove this

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 2]
```diff
@@ -35,6 +37,7 @@ cluster_groups:
   - build05
   - build08
   - multi01
+  - vsphere-build02
```
Comment (bear-redhat) [inline core-services/ci-secret-bootstrap/_config.yaml:40]: remove this

File: ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml [hunk 3]
```diff
@@ -93179,3 +93179,43 @@ periodics:
         - key: sa.config-updater.multi01.config
           path: kubeconfig
         secretName: config-updater
+- agent: kubernetes
```
Comment (bear-redhat) [inline ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml:93182]: remove this

**Review by bear-redhat — CHANGES_REQUESTED** [review]

**Comment by bear-redhat** [comment]
/cc @hongkailiu 
The PR looks good to me now. Can you take another look?

**Comment by hongkailiu** [comment]
Could we please name the cluster `vsphere02`? Simple and less-confusing with `build02`.

**Comment by jcpowermac** [comment]
@hongkailiu renamed

**Comment by jcpowermac** [comment]
@bear-redhat @hongkailiu anything else?

should `pull-ci-openshift-release-master-vsphere02-dry` do anything yet?

**Comment by jcpowermac** [comment]
@bear-redhat can you check the secret please. It still doesn't seem to be available.
```
The pod could not start because it could not mount the volume "build-farm-credentials": references non-existent secret key: sa.config-updater.vsphere02.config
```

**Comment by jcpowermac** [comment]
@hongkailiu @bear-redhat please review, see errors with secret.

**Comment by jmguzik** [comment]
@jcpowermac this PR is causing our dptp-controller-manager to fail. It also failed on postsubmits. I'll try to revert it.
cc @bear-redhat @hongkailiu

**Pull request merged**